tests/E2E/test/campaigns/install_upgrade/02_autoupgrade.js

```diff
@@ -68,11 +68,7 @@ scenario('The shop installation', () => {
     test('should set the Number of the version you want to upgrade to', () => client.waitAndSetValue(ModulePage.version_number, global.filename.replace(".zip", "")));
     test('should click on "save" button', () => client.waitForExistAndClick(ModulePage.save_button));
     test('should verify the success message', () => client.waitForVisibleElement(ModulePage.save_message));
-    test('should click on "Upgrade PrestaShop now!" button', () => {
-      return promise
-        .then(() => client.scrollTo(ModulePage.upgrade_block))
-        .then(() => client.waitForExistAndClick(ModulePage.upgrade_button, 50, 7000));
-    });
+    test('should click on "Upgrade PrestaShop now!" button', () => client.waitForExistAndClick(ModulePage.upgrade_button));
```

Comment:
How is that block of code less prone to errors than scrolling to the upgrade block before clicking ?
We need to be sure that it works in a small screen too.

Reply:
Yes, I tested with the small screen and now it works fine, the test is red with the scrollTo that's why I delete the line.
And also there is a mistake in `waitForExistAndClick` there is a marge of 50 in the place of the timeout